src/utils/constants.js

```diff
@@ -0,0 +1,7 @@
+export const links = {
+  veganHacktivistsHome: "https://veganhacktivists.org",
```

Comment:
Perhaps a difference in code style, but I feel like these values don't necessarily have to be taken apart into their own file. It is more code, there's more indirection, and I don't really see the benefit (if I'm missing something, please let me know!).

Reply:
The main benefit I thought was we have links, like katerodman, that are used in multiple places (home page and footer). So instead of reproducing them in multiple files we could just have one central location for links. If we ever need to change a link we have just one place to check instead of updating multiple component files. Your final call :)

Reply:
For now I think it's fine 👍